engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.rendering.dag;
```

Comment:
I am fond of shorter, simple and clean names. But I am not sure about how module developers interact with the existing nodes. So can we decide this in future? _(I can a put a comment to Node interface for remembering this.)_

Thanks for the making PRs "cleaner" advice. I will keep this in mind. I thought "reviewers" usually just check whole diff and give feedback accordingly, not going throughout every individual commit.